index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alfred Portfolio Website</title>
+  </head>
+  <body>
+    <main class="main">
+      <header class="header">
+        <a class="header-logo" href="#">Alfred</a>
+        <a href="javascript:void(0);" class="header-humbugger-icon">
+          <i class="fa fa-bars"></i>
+        </a>
+      </header>
+      <section class="developer">
+        <div class="developer-info">
+          <img
+            class="developer-info-shadow"
+            src="./images/header@2x.png"
+            alt=""
+          />
+          <p class="developer-info-name">Laura Jong</p>
+          <p class="developer-info-title">Software Developer</p>
```

Comment:
- [ ] Kindly make the main name a h1 tag and the subtitle any hx tag either h2-h6 tag